src/components/Card.js

```diff
@@ -20,15 +19,14 @@ export default class Card {
     this._isLiked = isLiked;
     this._handleDeleteClick = handleDeleteClick;
     this._handleLikeClick = handleLikeClick;
-
   }
 
   _setEventListeners() {
     //".card__like-button"
     this._cardElement
       .querySelector(".card__like-button")
```

Comment:
You've found `this._likeButton` in line 81 and it's accessible here


```js
    this._likeButton.addEventListener("click", () => {



```


`Class fields` (the so called **class variables** with `this`) are accessible inside any method of the class. You shouldn’t pass **class fields** into the call of any method because you can use class fields inside all methods.